### PR TITLE
Bump Django versions to 2.2.1

### LINKED ISSUE #76: Upgrade Django to 2.2.1
See:

- https://www.djangoproject.com/weblog/2019/may/01/django-bugfix-release-221/
- https://docs.djangoproject.com/en/2.2/releases/2.2.1/
- https://travis-ci.org/azavea/docker-django/builds/514789639

### PR DESCRIPTION
**Pull request by rbreslow**
## Overview

Update Django 2.2.x.

See:

 * https://docs.djangoproject.com/en/2.2/releases/2.2.1/

Resolves #76 

## Testing

See CI checks.

### CODE REVIEW

File: Dockerfile-alpine.template [hunk 1]
```diff
@@ -9,12 +9,12 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
-        --repository http://dl-cdn.alpinelinux.org/alpine/v3.9/main \
+        --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```
Comment (hectcastro) [inline Dockerfile-alpine.template]: I had to v3.8 to v3.9 for the addition of Django 2.2.x to the matrix. Do you know why `edge` has to be used here?

File: 2.2/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 gunicorn==19.9.0
-django==2.2
-psycopg2==2.7.7 --no-binary psycopg2
+django==2.2.1
+psycopg2==2.8.2 --no-binary psycopg2
```
Comment (hectcastro) [inline 2.2/requirements.txt]: It is possible that the `--no-binary` option is no longer necessary post 2.7.x.

See: http://initd.org/psycopg/docs/install.html

File: Dockerfile-alpine.template (hunk 1 above)
Comment (rbreslow) [reply Dockerfile-alpine.template]: ~~I think they were taken out of `edge` when we did https://github.com/azavea/docker-django/commit/6a5baa8d1c2a609dccd61c43952811624b83fc37, but now there are packages in `edge` that are more up-to-date than the packages in `v3.9`.~~

The packages in `edge` are more up-to-date than the packages in `v3.9`. I could go either way.

File: 2.2/requirements.txt (hunk 2 above)
Comment (rbreslow) [reply 2.2/requirements.txt]: Nice find. Good instinct to look a bit further too.

File: Dockerfile-alpine.template (hunk 1 above)
Comment (hectcastro) [reply Dockerfile-alpine.template]: I like not using `edge`. Mostly because I'm worried about transitive dependency resolutions sneaking in from there.

**Review by hectcastro — APPROVED** [review]
👍

**Pull request merged**